bundles/org.openhab.binding.souliss/README.md

```diff
@@ -0,0 +1,350 @@
+# Souliss Binding
+
+[Souliss](http://www.souliss.net/) is a networking framework for Arduino and compatibles boards, and is designed to let you easily build a smart home that is distributed over multiple boards via Ethernet, WiFi, wireless point-to-point and RS485 bus. 
+
+Souliss is an open-source and community driven project, you can use the [wiki](https://github.com/souliss/souliss/wiki) and [Community](https://github.com/souliss/souliss/wiki/Community) to get help and share your results.  
+
+## Prerequisites
+
+The binding requires a deployed network.  As a minimum, you need one Souliss node with Ethernet or WiFi access configured as a [Gateway](https://github.com/souliss/souliss/wiki/Gateway). A Gateway is a special node that is able to communicate with the user interfaces. The binding interacts as a user interface for Souliss.
```

Comment:
```suggestion
The binding requires a deployed network. 
As a minimum, you need one Souliss node with Ethernet or WiFi access configured as a [Gateway](https://github.com/souliss/souliss/wiki/Gateway). 
A Gateway is a special node that is able to communicate with the user interfaces. 
The binding interacts as a user interface for Souliss.
```
Example of how each sentence now starts on its own line. Please do to the whole file.

Reply:
fixed